### ISSUE TITLE
Can't upload images on fresh install

### ISSUE THREAD

**Issue by PolarizedIons** [Pe46dro/XBackBone-docker#14]
Basic command, plus some stuff to work with traefik. This happens whether I mount the mountpoints to my system or not.

```bash
docker run 
    -d \
    --network=traefik \
    -l traefik.enable=true \
    -l traefik.http.routers.xbackbone.rule="Host(\`ss.polarizedions.net\`)" \
    -l traefik.http.services.xbackbone.loadbalancer.server.port=80 \
    -e APP_NAME=XBackBone \
    -e URL=https:\/\/ss.polarizedions.net \
    --name xbackbone \
    pe46dro/xbackbone-docker:latest
```

On the UI it just says "Slim Application Error"

Log file:
> [2020-04-15 19:18:07] app.INFO: User admin logged in. [] []
> [2020-04-15 19:18:23] app.ERROR: Slim Application Error
> Type: League\Flysystem\Exception
> Code: 0
> Message: Impossible to create the root directory &quot;/app/storage/GIli0&quot;. mkdir(): Permission denied
> File: /app/vendor/league/flysystem/src/Adapter/Local.php
> Line: 112
> Trace: #0 /app/vendor/league/flysystem/src/Adapter/Local.php(156): League\Flysystem\Adapter\Local->ensureDirectory('/app/storage/GI...')
> #1 /app/vendor/league/flysystem/src/Filesystem.php(87): League\Flysystem\Adapter\Local->writeStream('GIli0/JUZEKudI2...', Resource id #183, Object(League\Flysystem\Config))
> #2 /app/app/Controllers/UploadController.php(192): League\Flysystem\Filesystem->writeStream('GIli0/JUZEKudI2...', Resource id #183)
> #3 /app/app/Controllers/UploadController.php(62): App\Controllers\UploadController->saveMedia(Object(GuzzleHttp\Psr7\Response), Object(GuzzleHttp\Psr7\UploadedFile), Object(stdClass))
> #4 [internal function]: App\Controllers\UploadController->uploadWeb(Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response))
> #5 /app/vendor/php-di/invoker/src/Invoker.php(82): call_user_func_array(Array, Array)
> #6 /app/vendor/php-di/slim-bridge/src/ControllerInvoker.php(47): Invoker\Invoker->call(Array, Array)
> #7 /app/vendor/slim/slim/Slim/Routing/Route.php(381): DI\Bridge\Slim\ControllerInvoker->__invoke(Array, Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
> #8 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(81): Slim\Routing\Route->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #9 /app/app/Middleware/CheckForMaintenanceMiddleware.php(26): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #10 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(245): App\Middleware\CheckForMaintenanceMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\MiddlewareDispatcher))
> #11 /app/app/Middleware/AuthMiddleware.php(33): class@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #12 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(245): App\Middleware\AuthMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(class@anonymous))
> #13 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(81): class@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #14 /app/vendor/slim/slim/Slim/Routing/Route.php(341): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #15 /app/vendor/slim/slim/Slim/Routing/RouteRunner.php(84): Slim\Routing\Route->run(Object(GuzzleHttp\Psr7\ServerRequest))
> #16 /app/app/Middleware/InjectMiddleware.php(21): Slim\Routing\RouteRunner->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #17 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(245): App\Middleware\InjectMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\Routing\RouteRunner))
> #18 /[20app/app/Middleware/LangMiddleware.php(25): class@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #19 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(245): App\Middleware\LangMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(class@anonymous))
> #20 /app/app/Middleware/RememberMiddleware.php(38): class@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #21 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(245): App\Middleware\RememberMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(class@anonymous))
> #22 /app/bootstrap/app.php(105): class@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #23 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(283): Closure->{closure}(Object(GuzzleHttp\Psr7\ServerRequest), Object(class@anonymous))
> #24 /app/vendor/slim/slim/Slim/Middleware/RoutingMiddleware.php(60): class@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #25 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(140): Slim\Middleware\RoutingMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(class@anonymous))
> #26 /app/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(98): class@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #27 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(140): Slim\Middleware\ErrorMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(class@anonymous))
> #28 /app/vendor/slim/slim/Slim/MiddlewareDispatcher.php(81): class@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #29 /app/vendor/slim/slim/Slim/App.php(211): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #30 /app/vendor/slim/slim/Slim/App.php(195): Slim\App->handle(Object(GuzzleHttp\Psr7\ServerRequest))
> #31 /app/index.php(10): Slim\App->run()
> #32 {main}
> Tips: To display error details in HTTP response set "displayErrorDetails" to true in the ErrorHandler constructor. [] []

**Comment by Hiren-Z** [comment]
Did you fix the permissions? (bottom of the readme)

**Comment by PolarizedIons** [comment]
When I mounted to the system, yes, though I can't when running on a volume

**Comment by Pe46dro** [comment]
Fixed volume in ef3f5b21b1d3525005a82764a222cbd7344276de
Travis will test docker volume
